{"name": "Проверка, что пользователь не может авторизоваться с неверным логином", "status": "passed", "steps": [{"name": "Отправка POST-запроса на эндпоинт: 'auth/login'", "status": "passed", "parameters": [{"name": "endpoint", "value": "'auth/login'"}, {"name": "headers", "value": "None"}, {"name": "json", "value": "{'email': 'thylpmmiqm@test.rug', 'password': 'zwrzjtvkbz'}"}], "start": 1727670847712, "stop": 1727670848822}], "start": 1727670847712, "stop": 1727670848822, "uuid": "de20326b-953d-4fd9-bc5f-9a20e1a5294b", "historyId": "cb34b985631bce7c9cd0b616fc12ff03", "testCaseId": "cb34b985631bce7c9cd0b616fc12ff03", "fullName": "tests.test_login.TestLogin#test_login_user_error_nonexistent_email", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-DPIGL2N"}, {"name": "thread", "value": "28804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}